lib/config.ts

```diff
@@ -136,13 +137,13 @@ export interface Config {
    */
   sauceKey?: string;
   /**
-   * Use sauceAgent if you need custom HTTP agent to connect to saucelabs.com.
+   * Use sauceAgent if you need custom HTTP agent to connect to saucelabs.com APIs.
    * This is needed if your computer is behind a corporate proxy.
    *
    * To match sauce agent implementation, use
    * [HttpProxyAgent](https://github.com/TooTallNate/node-http-proxy-agent)
-   * to generate the agent or use webDriverProxy as an alternative. If a
-   * webDriverProxy is provided, the sauceAgent will be overridden.
```

Comment:
Sure. I remember trying to set it all to webDriverProxy at one point 3 years ago...